stable_plugins/classical_ml/data_preparation/pandas_preprocessing/routes.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2023 QHAna plugin runner contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from tempfile import NamedTemporaryFile
+from http import HTTPStatus
+from typing import Mapping, Optional
+
+from pathlib import Path
+
+from celery.canvas import chain
+from flask import send_file, Response, redirect, Markup
+from flask.globals import request
+from flask.helpers import url_for
+from flask.templating import render_template
+from flask.views import MethodView
+from marshmallow import EXCLUDE
+
+from . import PDPreprocessing_BLP, PDPreprocessing
+from .schemas import (
+    FirstInputParametersSchema,
+    SecondInputParametersSchema,
+    TaskResponseSchema,
+)
+from qhana_plugin_runner.api.plugin_schemas import (
+    DataMetadata,
+    EntryPoint,
+    PluginMetadata,
+    PluginMetadataSchema,
+    PluginType,
+    InputDataMetadata,
+)
+from qhana_plugin_runner.db.models.tasks import ProcessingTask
+from qhana_plugin_runner.tasks import add_step, save_task_error, save_task_result
+
+from .tasks import first_task, second_task
+
+from celery.utils.log import get_task_logger
+
+TASK_LOGGER = get_task_logger(__name__)
+
+
+@PDPreprocessing_BLP.route("/")
+class PluginsView(MethodView):
+    """Plugins collection resource."""
+
+    @PDPreprocessing_BLP.response(HTTPStatus.OK, PluginMetadataSchema)
+    @PDPreprocessing_BLP.require_jwt("jwt", optional=True)
+    def get(self):
+        """pandas-preprocessing endpoint returning the plugin metadata."""
+
+        return PluginMetadata(
+            title="Pandas Preprocessing",
+            description=PDPreprocessing.instance.description,
+            name=PDPreprocessing.instance.identifier,
```

Comment:
```suggestion
            name=PDPreprocessing.instance.name,
```
See UST-QuAntiL/qhana-plugin-registry#27